ticdc/ticdc-debezium.md

```diff
@@ -25,30 +31,421 @@ In addition, the original Debezium format does not include important fields such
 
 ## Message format definition
 
-This section describes the format definition of the DML event output in the Debezium format.
+This section describes the message formats of DDL events, DML events and WATERMARK events.
 
-### DML event
+### DDL event
 
-TiCDC encodes a DML event into a Kafka message, with both the key and value encoded in the Debezium format.
+TiCDC encodes a DDL event into a Kafka message, with both the key and value encoded in the Debezium format.
 
-### Key format
+#### Key format
 
 ```json
 {
     "payload": {
-        "a": 4
+        "databaseName": "test"
     },
     "schema": {
+        "type": "struct",
+        "name": "io.debezium.connector.mysql.SchemaChangeKey",
+        "optional": false,
+        "version": 1,
         "fields": [
             {
-                "field": "a",
+                "field": "databaseName",
+                "optional": false,
+                "type": "string"
+            }
+        ]
+    }
+}
+```
+
+The fields in the key only include the database name. The fields are explained as follows:
+
+| Field            | Type    | Description                                         |
+|:------------------|:--------|:------------------------------------------------|
+| `payload`        | JSON    | The information about database name. |
```

Comment:
```suggestion
| `payload`        | JSON    | The database name. |
```

Reply:
Since it's JSON type, I'll keep the original wording.